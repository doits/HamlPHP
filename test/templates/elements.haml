!!! 5
%html
    %body
        - if ($image != null)
            %img{:src => $image}/
        - else
            Image not found
        
        = 'Some other message'