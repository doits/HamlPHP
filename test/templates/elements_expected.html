<!DOCTYPE html>
<html>
    <body>
        <?php if ($image != null): ?>
            <img <?php atts(array('src' => $image)); ?> />
        <?php else: ?>
            Image not found
        <?php endif; ?>
        <?php echo 'Some other message' ?>
    </body>
</html>
